main/src/modules/reactiveLayer/behaviorManager/src/pointing.cpp

```diff
@@ -8,28 +8,50 @@ void Pointing::configure() {
 }
 
 void Pointing::run(Bottle args/*=Bottle()*/) {
-    yInfo() << "Pointing::run";
-    Bottle *sensation = sensation_port_in.read();
-    
-    int id = yarp::os::Random::uniform(0, sensation->size()-1);
-    yDebug()<<"Randomly selected: "<< id; // should be random here 
-    string obj_name = sensation->get(id).asList()->get(1).asString();
-    
-    iCub->say("I could point to the " + obj_name);
-    Time::delay(2.0);
-    // yInfo() << "About to point the " + obj_name;
-    // return;
-
-    iCub->say("Do you know that this is a " + obj_name);
-    Object* obj = iCub->opc->addOrRetrieveEntity<Object>(obj_name);
-    string sHand = "right";
-    if (obj->m_ego_position[1]<0) 
-        sHand = "left";
-
-    Bottle bHand(sHand); 
-
-    bool succeeded = iCub->point(obj_name, bHand);
-    Time::delay(0.5);
+	yInfo() << "Pointing::run";
+	Bottle *sensation = sensation_port_in.read();
+
+	int id = yarp::os::Random::uniform(0, sensation->size() - 1);
+	yDebug() << "Randomly selected: " << id; // should be random here 
+	string obj_name = sensation->get(id).asList()->get(1).asString();
+
+	iCub->say("I could point to the " + obj_name);
+	Time::delay(2.0);
+	// yInfo() << "About to point the " + obj_name;
+	// return;
+
+	Object* obj = iCub->opc->addOrRetrieveEntity<Object>(obj_name);
+	string sHand = "right";
+	if (obj->m_ego_position[1] < 0)
+		sHand = "left";
```

Comment:
Cool.
@clement-moulin-frier @sockchinglow @jypuigbo can you please change your modules to not pass `bHand` to the `ARE` subsystem? That shortens the code by quite a bit.

Thanks,
Tobi